.config/constraints.txt

```diff
@@ -16,6 +16,7 @@ charset-normalizer==3.3.2
 click==8.1.7
 colorama==0.4.6
 coverage==7.5.1
+coverage-enable-subprocess==1.0
```

Comment:
Based on my experience this extension is not needed, a more reliable way to achieve this is just to ensure that the environment variable used by coverage is passed to subprocesses, like we do on linter (see tox.ini file).